great_expectations/datasource/fluent/sql_datasource.py

```diff
@@ -921,14 +921,19 @@ def test_connection(self) -> None:
                 f'"{self.schema_name}" does not exist.'
             )
 
-        if not inspector.has_table(
-            table_name=self.table_name,
-            schema=self.schema_name,
-        ):
-            raise TestConnectionError(
-                f"Attempt to connect to table: {self.qualified_name} failed because the table"
-                f" {self.table_name} does not exist."
+        try:
+            with engine.connect() as connection:
+                connection.execute(
+                    sa.text(f"SELECT * FROM {self.qualified_name} LIMIT 1;")
```

Comment:
Will we be able to perform a select here since a datasource doesn't know about a given table like a data asset would?

Reply:
Can we use `sa.select` here?

Reply:
Done